packages/lexical-editor/src/plugins/WebinyListPLugin/WebinyListPlugin.ts

```diff
@@ -0,0 +1,21 @@
+import { useLexicalComposerContext } from "@lexical/react/LexicalComposerContext";
+import { useEffect } from "react";
+import { WebinyListNode } from "~/nodes/list-node/WebinyListNode";
+import { WebinyListItemNode } from "~/nodes/list-node/WebinyListItemNode";
+import { useWebinyList } from "~/hooks/useWebinyList";
```

Comment:
At the moment I will leave this for the polishing phase in favor of time because Lexical is sensitive to these kinds of changes. I need to test everything again.